src/pages/boundary/docs/[...page].tsx

```diff
@@ -0,0 +1,32 @@
+import boundaryData from 'data/boundary.json'
+import { ProductData } from 'types/products'
+import { getStaticGenerationFunctions } from 'views/docs-view/server'
+import DocsView from 'views/docs-view'
+import { GetStaticPathsContext, GetStaticPathsResult } from 'next'
+import { removeIndexPath } from 'lib/remove-index-path'
+
+const basePath = 'docs'
+const baseName = 'Docs'
+const product = boundaryData as ProductData
+
+const { getStaticPaths: generatedGetStaticPaths, getStaticProps } =
+	getStaticGenerationFunctions({
+		product,
+		basePath,
+		baseName,
+	})
+
+/**
+ * Wrapper for `generatedGetStaticPaths`. It handles removing the index path
+ * from the `paths` array returned by `generatedGetStaticPaths`.
+ */
+async function getStaticPaths(
```

Comment:
Have moved this from `[[...page]].tsx` to `[...page].tsx` in order to avoid conflict with the `src/pages/[productSlug]/docs/index.tsx` custom `/docs` landing route.